ldapadmin/src/main/java/org/georchestra/ldapadmin/ds/AccountDaoImpl.java

```diff
@@ -446,7 +447,16 @@ private void setAccountField(DirContextOperations context, String fieldName, Obj
 
         if (!isNullValue(value)) {
             context.setAttributeValue(fieldName, value);
-        }
+        } else {
+            Object[] values = context.getObjectAttributes(fieldName);
+            if(values != null){
+            if (values.length == 1) {
+                LOG.info("Removing attribue " + fieldName);
+                context.removeAttributeValue(fieldName, values[0]);
+            } else {
```

Comment:
Indeed, @vdorut please fix the indent (select the code block in eclipse, then ctrl+shift+f ;-))